src/pages/besquare/static/style/_hero.js

```diff
@@ -34,6 +36,6 @@ export const ImageWrapper = styled.img`
         max-width: 328px;
         margin-right: 15px;
         overflow: unset;
-        top: 40px;
+        top: 15px;
```

Comment:
```suggestion
        top: 24px;
```

Reply:
Can we change top to 24px as the gap between HeroContainer and ImageWrapper is 24px based on design.

Reply:
ok